dist/gmusic.js

```diff
@@ -211,13 +211,16 @@ proto.playback = {
   toggleShuffle: function () { this.playback._shuffleEl.click(); },
 
   getRepeat: function () {
-    var title = this.playback._repeatEl.getAttribute('title').toLowerCase();
-    if (title.indexOf('repeat off') !== -1) {
-      return GMusic.Playback.NO_REPEAT;
-    } else if (title.indexOf('repeating all') !== -1) {
+    // States possible:
+    // no-repeat: No class 'active', icon: av:repeat
+    // repeat: Has class 'active', icon: av:repeat
+    // repeat-one: Has class 'active', icon: av:repeat-one
+    if (this.playback._repeatEl.__data__.icon === 'av:repeat-one') {
```

Comment:
But at this point, we are more/less redundant to the code so maybe adjacent comments will suffice:

```js
// If repeat is inactive, return `NO REPEAT`
// DEV: `NO_REPEAT` shares same repeat icon as `LIST_REPEAT`
//   el.__data__.icon ==='av:repeat'
if (!this.playback._repeatEl.classList.contains('active')) {
  return GMusic.Playback.NO_REPEAT;
// Otherwise if we are repeating once, return `SINGLE_REPEAT`
} else if (this.playback._repeatEl.__data__.icon === 'av:repeat-one') {
  return GMusic.Playback.SINGLE_REPEAT;
// Otherwise, return `LIST_REPEAT`
} else {
  return GMusic.Playback.LIST_REPEAT;
}
```